test/matchers.test.ts

```diff
@@ -50,6 +49,6 @@ test('matchers', () => {
         'toBeRequested',
         'toBeRequestedTimes',
         'toBeRequestedWith',
-        'toBeRequestedWithResponse'
+        'toBeRequestedWithResponse',
```

Comment:
this change doesn't seem to be necessary
```suggestion
        'toBeRequestedWithResponse'
```